File: php/kasir.php
<?php
echo "Masukkan nama barang: ";
$nama = trim(fgets(STDIN));

echo "Masukkan harga barang: ";
$harga = floatval(trim(fgets(STDIN)));

echo "masukkan jumlah beli: ";
$jumlah = intval(trim(fgets(STDIN)));

$total =$harga * $jumlah;

if ($total > 56000){
    $diskon = $total * 0.06;
} else {
    $diskon = 0.0;
}

$total_bayar = $total - $diskon;

    echo "\n===== STRUK PEMBELIAN =====\n";
    echo  "Nama barang : $nama\n";
    echo "Harga Satuan : $harga\n";
    echo  "Jumlah Beli : $jumlah \n";
    echo  "Total Harga : $total\n";
    echo  "Diskon : $diskon \n";
    echo  "Total bayar : $total_bayar\n";
    echo  "========================\n";
    ?>

File: python/kasir.py
nama = input("Masukkan nama barang: ")
harga = float(input("Masukkan harga barang: "))
jumlah = int(input("Masukkan jumlah beli: "))

total = harga * jumlah

if total > 56000:
    diskon = total = 0.06
else:
    diskon =0
    
total_bayar = total - diskon

print("\n===== STRUK PEMBELIAN =====")
print(f"Nama barang : {nama}")
print(f"Harga Satuan : {harga}")
print(f"jumlah Beli : {jumlah}")
print(f"Total Harga : {total} ")
print(f"Diskon : {diskon}")
print(f"Total bayar : {total_bayar}")
print("=============================")